lib/lookup.json

```diff
@@ -2,7 +2,7 @@
   "@hapi/hapi": {
     "prefix": "v",
     "maintainers": ["devinivy", "DavidTPate", "AdriVanHoudt"],
-    "skip": ["osx", "aix"]
+    "skip": ["darwin", "aix", "debian-10"]
```

Comment:
I suspect that `debian-10` should be `debian10` to skip on Debian 10.

Reply:
Oh okay - I'll make a follow up. I went back in the git history, and the only reference I could find to debian was with a hyphen.

Reply:
Maybe we can add a test that all the os names in lookup have correct values?